test/src/module/command/infoTest.c

```diff
@@ -1120,7 +1121,8 @@ testRun(void)
             "            database size: 19.2MB, backup size: 8.2KB\n"
             "            repository size: 2.3MB, repository backup size: 346B\n"
             "            backup reference list: 20181119-152138F, 20181119-152138F_20181119-152152D\n"
-            "            database list: mail (16456), postgres (12173)\n",
+            "            database list: mail (16456), postgres (12173)\n"
+            "            files with page checksum error: base/16384/17000\n",
```

Comment:
This test should show output with more than one file, so in line 1094 modify the code so that there are 2 page checksum errors.  This will mean the test at line 1069 tests 1 error, this one at line 1125 tests more than one error and, given the comment below, then the one at line 1177 would test no errors.

Reply:
Thanks for your review. I applied the tests structure you advised.

Tests are running fine indeed then:

```
2020-11-05 09:25:52.108 P00   INFO: P1-T1/1 - vm=u18, module=command, test=info (9.16s)
2020-11-05 09:25:53.065 P00   INFO: tested modules have full coverage
2020-11-05 09:25:53.065 P00   INFO: writing C coverage report
2020-11-05 09:25:53.189 P00   INFO: TESTS COMPLETED SUCCESSFULLY (11s)
```